plugins/com.python.pydev.analysis/src/com/python/pydev/analysis/mypy/MypyAnalysis.java

```diff
@@ -54,6 +54,68 @@
     private IDocument document;
     private IPath location;
 
+    private static class LineCol {
+        @Override
```

Comment:
Nitpick:

Please move the hashCode and equals methods to be after the constructor.